Remove empty comment for db table, as it causes an issues with db schema inconsistency - database creates field with NULL comment instead of empty string.

diff --git a/etc/db_schema.xml b/etc/db_schema.xml
index bedcac1..c88c333 100644
--- a/etc/db_schema.xml
+++ b/etc/db_schema.xml
@@ -149,7 +149,7 @@
         </constraint>
     </table>
     <table name="mailchimp_notification">
-        <column xsi:type="int" name="id" padding="10" nullable="false" identity="true" comment=""/>
+        <column xsi:type="int" name="id" padding="10" nullable="false" identity="true"/>
         <column xsi:type="text" name="notification_data" nullable="false" comment="data of the request"/>
         <column xsi:type="datetime" name="generated_at" on_update="false" nullable="false" default="CURRENT_TIMESTAMP"
                 comment="date of generation"/>
